cmd/testcase_update_test.go

```diff
@@ -24,7 +24,7 @@ func TestPrintValidationResultHuman_ValidationErrors(t *testing.T) {
 			{Code: "E0", Title: "Validation Error"},
 		},
 	})
-	assert.Equal(t, buf.String(), "# FILE: test.js\nWARN: TestCase updated, but validation errors occured.\n\n1) E0: Validation Error\n\n")
+	assert.Equal(t, buf.String(), "WARN: TestCase updated, but validation errors occured.\n\n1) E0: Validation Error\n")
```

Comment:
For non-eval errors and for `test-case validate` we could run into problems if we validate multiple test cases via `forge test-case validate *.js`. In this case we need to know which file did contain the issues.

Reply:
I opted to add the filename output only to the `forge test-case validate`. I don't think the filename adds much information to the other commands.

Reply:
Yeah. In all other cases it can only reference one "file".